packages/x-components/src/views/home/Home.vue

```diff
@@ -86,13 +86,20 @@
           >
             <BaseKeyboardNavigation>
               <div class="x-input-group x-input-group--card">
+                <SearchInputPlaceholder
+                  :messages="searchInputPlaceholderMessages"
+                  :animate-only-on-hover="true"
+                >
+                  Search
+                </SearchInputPlaceholder>
                 <SearchInput
                   aria-label="Search for products"
-                  placeholder="Search"
                   :instant="controls.searchInput.instant"
                   :instant-debounce-in-ms="controls.searchInput.instantDebounceInMs"
                 />
-                <ClearSearchInput aria-label="Clear query">Clear</ClearSearchInput>
+                <ClearSearchInput v-if="$x.query.searchBox" aria-label="Clear query">
```

Comment:
this `v-if` is breaking the E2E tests. Because we clear the query from some tests by clicking this button. If the button is not visible because there is no query, the test fails because it does not find the button.
Even if there is no query in the tests we are clicking it just in case to make sure there is no query in the input.